src/main/java/command/Command.java

```diff
@@ -0,0 +1,33 @@
+package command;
+import components.Storage;
+import components.Ui;
+import task.TaskList;
+import exception.NyabotException;
+
+
+/**
+ * Represents command from user input to execute.
+ */
+abstract public class Command {
+
+    protected boolean isExit = false;
+
+    /**
+     * Returns whether the command is an exit command.
+     *
+     * @return Boolean whether the command is an exit command.
+     */
+    public boolean isExit() {
```

Comment:
Perhaps can consider renaming this method to a verb for clarity or to avoid confusion with boolean variable isExit?